pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/Microsoft.NETCore.UniversalWindowsPlatform.depproj

```diff
@@ -27,28 +27,20 @@
     <ItemGroup  Condition="'$(NuGetRuntimeIdentifier)' != ''">
       <_runtimeCLR Include="@(ReferenceCopyLocalPaths)" Condition="'%(ReferenceCopyLocalPaths.FileName)' == 'coreclr' OR '%(ReferenceCopyLocalPaths.FileName)' == 'libcoreclr'" />
     </ItemGroup>
-    <!-- start with all the files resolved -->
-    <PropertyGroup Condition="'@(_runtimeCLR)' != ''">
-      <_runtimePackageId>%(_runtimeCLR.NuGetPackageId)</_runtimePackageId>
-      <_runtimePackageVersion>%(_runtimeCLR.NuGetPackageVersion)</_runtimePackageVersion>
-      <_runtimePackagePath>$(PackagesDir)/$(_runtimePackageId)/$(_runtimePackageVersion)</_runtimePackagePath>
-    </PropertyGroup>
-
     <ItemGroup Condition="'$(NuGetRuntimeIdentifier)' != ''">
       <!-- RID-specific: include all runtime files -->
       <_FilesToPackage Include="@(ReferenceCopyLocalPaths)">
         <!-- ResolveNugetPackageAssets doesn't preserve the asset type (native), 
              calculate it by looking for native in the path -->
         <IsNative Condition="$([System.String]::new('%(Identity)').ToLowerInvariant().Replace('\', '/').Contains('/native/'))">true</IsNative>
       </_FilesToPackage>
+      <!-- Exclude System.IO.Compression files as they get pulled into the closure by WCF -->
```

Comment:
Actually I think I understand what's going on here.  We previously shipped a clrcompression.dll in the lib folder.  https://github.com/dotnet/corefx/commit/d0ae14f6136fbc4b3af428b71523758c983c24f0

This means that the IsNative condition is actually the problem https://github.com/dotnet/core-setup/pull/1967/commits/1a99f0846db37aac669c4f023314d9b9e7ca80b1#diff-e5fdfbf0dc7f5037aa5538f270da8d8eR35

What I'd do is add another line above.
```
<IsNative Condition="'%(FileName)%(Extension)' == 'clrcompression.dll'">true</IsNative>

```